shared/menubar/files.desktop.js

```diff
@@ -121,13 +120,7 @@ const ComposedFileUpdates = FileUpdatesHoc(FileUpdates)
 const UserTlfUpdateRow = (props: UserTlfUpdateRowProps) => (
   <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.tlfRowContainer}>
     <Kb.ClickableBox onClick={props.onSelectPath}>
-      <PathItemIcon
-        path={props.path}
-        size={32}
-        type="folder"
-        username={props.username}
-        style={styles.tlfRowAvatar}
-      />
+      <Kb.Avatar size={32} username={props.writer} style={styles.tlfRowAvatar} />
```

Comment:
Should the clickable change to user's profile instead then ?

Reply:
Done